$NetBSD: patch-at,v 1.1 2005/12/01 18:13:24 wiz Exp $

The info files are shipped pre-built and carry no directory entry, so
install-info has nothing to register and they never appear in the dir
node.  Add the block it looks for.

This is one of twenty identical additions, one per shipped info file.

--- info/R2L.orig	1994-10-27 12:34:29.000000000 +0100
+++ info/R2L
@@ -3,7 +3,10 @@ produced by `texinfo-format-buffer'
 from file `R2L.texi'
 using `texinfmt.el' version 2.32 of 19 November 1993.
 
-
+INFO-DIR-SECTION Editors
+START-INFO-DIR-ENTRY
+* R2L: (R2L).		Right-to-left writing on Mule.
+END-INFO-DIR-ENTRY
 
 
 File: R2L, Node: Top, Up: (mule)
